quupe-RN/js/screens/SignUp/SignUpContainer.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import SignUp from './SignUp';
+
+class SignUpContainer extends Component {
+  constructor() {
+    super();
+    this.state = {
```

Comment:
Nice work! Since we have our GraphCool server working for login and auth, let's use it instead of storing state in a component.